packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -0,0 +1,66 @@
+import { autorun } from 'mobx'
+import { addDisposer, getRoot, types } from 'mobx-state-tree'
+import _ from 'lodash'
```

Comment:
`import clone from 'lodash/clone'` for tree shaking

Reply:
Where MST has methods for acting on models, should we prefer those over lodash in the feedback code? Not that I know if MST has methods to clone objects.

Reply:
I got rid of this by using an observable map for feedback rules, rather than a frozen object.